server/src/server.ts

```diff
@@ -255,6 +259,14 @@ async function validatePerlDocument(textDocument: TextDocument): Promise<void> {
         nLog("Perl Critic Time: " + (Date.now() - start)/1000 + " seconds", settings);
         sendDiags({ uri: textDocument.uri, diagnostics: allNewDiags });
     }
+
+    const diagImports = await pImports;
+    documentDiags.set(textDocument.uri, diagImports); // May need to clear out old ones if a user changed their settings.
+    if(settings.perlimportsTidyEnabled){
+        const allNewDiags = perlOut.diags.concat(diagImports);
+        nLog(`perlimports Time: ${Date.now() - start) / 1000} seconds`, settings);
```

Comment:
Missing left paren

Reply:
Thanks for catching that! I thought my editor would have notified me. :(